working/declaring-constructors/feature-specification.md

```diff
@@ -363,10 +540,19 @@ a *very* small number of authors use.)
 So while the potential for confusion is there, I think it's unlikely to be a
 problem in practice.
 
+### Private field parameters and initializing formals
+
+Any existing initializing formals with private names must be positional since
+it's a compile-time error to have a private named parameter. Since those
+arguments are passed positionally, the change to give the parameter a public
+name has no effect on any callsites.
+
+Generated documentation may change, but that should be harmless.
```

Comment:
True. It won't break.
If we have `C(this._foo)` today, documentation saying `[_foo]` will now refer to the field, not to the parameter. It won't become an error, and it does point to *something* relevant, but it might be worth telling people that they should migrate to `[foo]` to refer to the parameter.

(Or DartDoc can be lenient and let both `[_foo]` and `[foo]` refer to the parameter, and show them both as `foo` in the output. That could be the starting point, then we can start warning about `[_foo]` and quick-fixing it when `[foo]` starts working. Maybe make `[_foo]` go back to referring to the field in Dart 4.0.)